cesql/spec.md

```diff
@@ -353,6 +357,25 @@ left operand of the OR operation evalues to `true`, the right operand MUST NOT b
 
 #### 3.7. Type casting
 
+A CESQL engine MUST support the following type casts:
+
+| Definition           | Semantics                                                                                                                                                                          |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Integer -> String`  | Returns the string representation of the integer value in base 10. If the value is less than 0, the '-' character is prepended to the result.                                      |
+| `String -> Integer`  | Returns the result of interpreting the string as a 32 bit base 10 integer. The string may begin with a leading sign '+' or '-'. If the result will overflow, an error is returned. |
```

Comment:
We may not want to worry about it, but are leading zeroes permitted? "0001" for example? (And if so, does that mean the string can be really long, e.g. "00000000000000000000000000000000001"? Leading zeroes foil length checks sometimes...)

Reply:
Personally, I think leading zeroes are fine - we aren't using any length checks on the input strings afaik, rather we validate the parsed integer value